tests/unit/h_vialib/client_test.py

```diff
@@ -24,15 +24,17 @@ class TestViaClient:
         "via.client.requestConfigFromFrame.origin": "http://random.localhost",
         "via.client.requestConfigFromFrame.ancestorLevel": "2",
         "via.external_link_mode": "new-tab",
+        "via.sec": Any.string(),
```

Comment:
This isn't the default options any more with the addition of the "via.sec", it's just the values we expect in a specific test. So we can get around the naming problem by just inlining it into the test where it's used.